packages/core/src/indexing/postgres/__tests__/postgres-index-api.test.ts

```diff
@@ -42,21 +42,22 @@ export async function listMidTables(dbConnection: Knex) {
   return dataArr
 }
 
+export async function dropMidTables() {
+  await dbConnection.schema.dropTableIfExists(STREAM_ID_A)
+  await dbConnection.schema.dropTableIfExists(STREAM_ID_B)
+}
+
 afterAll(async () => {
   await pgTeardown()
 })
 
-export async function dropMidTables(dbConnection: Knex) {
-  dbConnection.raw(`DROP TABLE IF EXISTS ${STREAM_ID_A};DROP TABLE IF EXISTS ${STREAM_ID_B};`)
-}
-
 describe('init', () => {
   describe('create tables', () => {
     test('create new table from scratch', async () => {
       const modelsToIndex = [StreamID.fromString(STREAM_ID_A)]
       const indexApi = new PostgresIndexApi(dbConnection, modelsToIndex)
       await indexApi.init()
-      const created = await listMidTables(dbConnection)
+      const created = await listMidTables()
       const tableNames = modelsToIndex.map((m) => `${m.toString()}`)
```

Comment:
```suggestion
      const tableNames = modelsToIndex.map(String)
```

Reply:
```suggestion
      const tableNames = modelsToIndex.map(asTableName)
```

Better be consistent.